cypress-io/cypress
# feat: output video file path on one row during run mode

- Closes #23913 

### User facing changelog
Moves video output link to it's own line in `run` mode to make the video output link more easily clickable in the terminal.

### Additional details
Improves the user experience to allow users to directly click the recorded link inside the terminal or copy/paste from a CI terminal more easily

### Steps to test
Easiest way to see changes locally is to run a single test in `cypress run` mode. For omitting values under `system-test`s in the core repo, please run the system tests and compare the snapshot.

### How has the user experience changed?

#### Before
https://user-images.githubusercontent.com/3980464/213530519-059c7d55-17b9-4b3b-b50c-b192253fd3bb.mp4

#### After
https://user-images.githubusercontent.com/3980464/213529911-2e685005-9b43-44a0-ba31-15d1b9db424b.mp4





### PR Tasks
